Fix by converting to bash.

mtbentley opened the pull request.

Commit:
Fix by converting to bash.

raidancampbell commented:
Needs more feature creep.  Write the valid MACs to a file with proper `DE:AD:DE:AF:BE:EF:` format, encoded in base64.  If the file exists on future runs, just plop out a random MAC from the list.

With this feature, I think it may literally be the same as my existing version

Commit:
Fixed to match comment on pull #1
With no params prints formatted mac addresses to stdout
With $1="full" outputs <ip>: <host> to stdout

mtbentley commented:
Fixed

Commit:
Add MAC verification via regex, add max threads via CPUS env var

Commit:
Output info to stderr

raidancampbell commented:
damnit.
